static/js/vue-cdr-access/src/mixins/fullRecordUtils.js

```diff
@@ -87,6 +86,28 @@ export default {
 
         toggleMetadata(show) {
             this.showMetadata = show;
+        },
+
+        fieldExists(value) {
+            return value !== undefined;
+        },
+
+        formatDate(value) {
+            return format(value, 'yyyy-MM-dd');
+        },
+
+        editDescriptionUrl(id) {
+            return `${window.location.host}/describe/${id}`;
+        },
+
+        hasAccess(permission, user_type= 'authenticated') {
+            const group_roles = this.recordData.briefObject.groupRoleMap;
```

Comment:
I thought the public UI wasn't returning the groupRole map? We can return it if we need to, or even produce an abbreviated version of it with the minimal permissions needed by the UI listed. We seem to be doing a bunch of these checks for patron permissions, so it seems like it might be necessary to return more than just whether there is edit description permission.

I would be happy to contribute some form of returning a reduced version of the role groups to this PR if that would be helpful

Reply:
The group roles map does get return. What it doesn't do is inherit permissions, so if the required permission for a field is `viewMetadata` and the group has `viewOriginals` access they're going to be able to view that field.